dao/src/main/java/com/carhouse/dao/mappers/CarMapper.java

```diff
@@ -66,7 +67,7 @@ public CarMapper(final FuelTypeMapper fuelTypeMapper, final TransmissionMapper t
     public Car mapRow(final ResultSet resultSet, final int i) throws SQLException {
         Car car = new Car(resultSet.getInt(CAR_ID),
                 resultSet.getDate(YEAR), resultSet.getInt(MILEAGE), fuelTypeMapper.mapRow(resultSet, i),
-                transmissionMapper.mapRow(resultSet, i), carModelMapper.mapRow(resultSet, i));
+                transmissionMapper.mapRow(resultSet, i), carModelMapper.mapRow(resultSet, i), null);
```

Comment:
look at setters with return type of the same class and the last line like `return this`. It might be helpful in your case.